packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/ssh-tunnel-tab.tsx

```diff
@@ -69,32 +75,64 @@ function SSHTunnel({
   connectionOptions,
   updateConnectionFormField,
   errors,
+  connectionStringUrl,
 }: {
   errors: ConnectionFormError[];
   connectionStringUrl: ConnectionStringUrl;
   updateConnectionFormField: UpdateConnectionFormField;
   connectionOptions?: ConnectionOptions;
 }): React.ReactElement {
-  const [selectedOption, setSelectedOption] = useState(options[0]);
+  const hasProxyHost = connectionStringUrl
+    .typedSearchParams<MongoClientOptions>()
+    .get('proxyHost');
+  const hasIdentityFile = connectionOptions?.sshTunnel?.identityKeyFile;
+  const hasPassword = connectionOptions?.sshTunnel?.password;
 
-  const optionSelected = useCallback((event: ChangeEvent<HTMLInputElement>) => {
-    event.preventDefault();
-    const item = options.find(({ id }) => id === event.target.value);
-    if (item) {
-      setSelectedOption(item);
-    }
-  }, []);
+  const seletedOptionType: SSHType = hasProxyHost
+    ? 'socks'
+    : hasIdentityFile
+    ? 'identity'
+    : hasPassword
+    ? 'password'
+    : 'none';
```

Comment:
I think we could default to 'password' to avoid sending user to NONE if there are options set. Do you see an issue with that? 

```
const selectedOptionType: SSHType = hasProxyHost
    ? 'socks'
    : !connectionOptions.sshTunnel
    ? 'none'
    : (hasIdentityFile || hasIdentityFilePassword)
    ? 'identity'
    : 'password';
```

Reply:
maybe this `!connectionOptions.sshTunnel` could be a bit more smart, something like: 

`const hasNoSshOptions = !sshTunnel || !Object.values(sshTunnel).find((v) => v)`